ts/components/leftpane/overlay/OverlayClosedGroup.tsx

```diff
@@ -197,7 +209,9 @@ export const OverlayClosedGroupV2 = () => {
           type="text"
           placeholder={window.i18n('groupNameEnter')}
           value={groupName}
-          onValueChanged={setGroupName}
+          onValueChanged={value => {
```

Comment:
This function should be separateand pass here? You can call it `onValueChanged` like `onEnterPressed`.

Reply:
fixed